server/src/routes/index.ts

```diff
@@ -1,19 +1,19 @@
 import { Router } from 'express';
 
+import authRouter from './auth.routes';
 import userRouter from './user.routes';
-import songRouter from './song.routes';
 import groupRouter from './group.routes';
 import setlistRouter from './setlist.routes';
 
 const getRoutes = (): Router => {
   const router = Router();
 
+  router.use('/auth', authRouter);
   router.use('/users', userRouter);
-  router.use('/songs', songRouter);
   router.use('/groups', groupRouter);
   router.use('/setlist', setlistRouter);
```

Comment:
actually now it bothers me that some of our route names are singular and some plural :') let's keep it consistent with singular?

Reply:
i think plural better